localstack/services/ses/provider.py

```diff
@@ -178,6 +181,29 @@ def create_configuration_set_event_destination(
 
         return result
 
+    @handler("DeleteConfigurationSet")
+    def delete_configuration_set(
+        self, context: RequestContext, configuration_set_name: ConfigurationSetName
+    ) -> DeleteConfigurationSetResponse:
+        # not implemented in moto
+        # TODO: contribute upstream?
+        backend = get_ses_backend(context)
+        backend.config_set.pop(configuration_set_name, None)
```

Comment:
question: I suppose AWS might return an exception if we try to delete a non-existent `ConfigurationSet`? It does not have the same behaviour for every services so I genuinely don't know.

Reply:
Thanks for the pointer - I'll create a follow up PR with error handling :+1: